src/Layer/C3DTilesLayer.js

```diff
@@ -325,12 +79,27 @@ class C3DTilesLayer extends GeometryLayer {
         /** @type {Style} */
         this._style = config.style || null;
 
-        /** @type {StyleController} */
-        this._styleController = new StyleController();
-
-        /** @type {Map<number, Map<number,C3DTileFeature>>} */
+        /** @type {Map<string,THREE.MeshStandardMaterial} */
+        this.#fillColorMaterialsBuffer = new Map();
+
+        /**
+         * Map all C3DTFeature of the layer according their tileId and their batchId
+         * Map< tileId, Map< batchId, C3DTFeature>>
+         *
+         * @type {Map<number, Map<number,C3DTFeature>>}
+         */
         this.tilesC3DTileFeatures = new Map();
 
+        if (config.onTileContentLoaded) {
+            console.warn('DEPRECATED onTileContentLoaded should not be passed at the contruction, use addEventListener instead');
```

Comment:
Could you modify the end by `use EVENT_TILE_CONTENT_LOADED event instead` please?